atomicfu-gradle-plugin/build.gradle.kts

```diff
@@ -40,3 +40,29 @@ tasks.processResources {
         expand("atomicfuVersion" to project.version)
     }
 }
+
+signing {
+    // disable signing if private key isn't passed
+    isRequired = findProperty("libs.sign.key.private") != null
+}
+
+gradlePlugin {
+    website.set("https://github.com/Kotlin/kotlinx-atomicfu")
+    vcsUrl.set("https://github.com/Kotlin/kotlinx-atomicfu.git")
+
+    plugins {
+        create("Atomicfu") {
+            id = "kotlinx-atomicfu"
+            implementationClass = "kotlinx.atomicfu.plugin.gradle.AtomicFUGradlePlugin"
+            displayName = "Gradle plugin for kotlinx-atomicfu library"
+            description = "Enables efficient use of atomic operations in Kotlin multiplatform projects."
+        }
+    }
+}
+
+// NOTE: `kotlinx-atomicfu.properties` file appears in the artifact twice, when publishing kotlin-atomicfu plugin using gradlePlugin{} block above
+tasks.withType<Copy>().named("processResources") {
```

Comment:
I'm not sure, if this is the best way to resolve the duplicate conflict 🤔:
```
Execution failed for task ':atomicfu-gradle-plugin:processResources'.
> Entry META-INF/gradle-plugins/kotlinx-atomicfu.properties is a duplicate but no duplicate handling strategy has been set. Please refer to https://docs.gradle.org/8.7/dsl/org.gradle.api.tasks.Copy.html#org.gradle.api.tasks.Copy:duplicatesStrategy for details.
```

Reply:
https://docs.gradle.org/current/userguide/java_gradle_plugin.html#ex-using-the-gradleplugin-block
Check the text below the example.
You should be able to delete the /atomicfu-gradle-plugin/src/main/resources/META-INF directory instead. As the gradlePlugin block is creating the files itself.

Reply:
Oh, cool, thank you! That solves the problem)